cmd/listen.go

```diff
@@ -1,14 +1,15 @@
 package main
 
 import (
-	"github.com/google/gopacket"
-	"github.com/google/gopacket/layers"
-	"github.com/google/gopacket/pcap"
-	log "github.com/sirupsen/logrus"
 	"net"
 	"strings"
 	"sync"
 	"time"
+
+	"github.com/google/gopacket"
```

Comment:
When I run `go fmt` or check in the vim go plugin, it doesn't do this and reverts to the original order